web/packages/operations/.env.example

```diff
@@ -15,8 +15,8 @@ REACT_APP_INFURA_KEY=
 # GRAPHQL config
 GRAPHQL_API_URL=https://data.snowbridge.network/graphql
 
-# Scan interval(in minutes)
-SCAN_INTERVAL=60 
+# Scan interval(30 minutes by default)
+SCAN_INTERVAL=30
```

Comment:
Do you think it would be valuable to do 15 minutes?

Reply:
I'd prefer to start from 30 minutes first.